packages/chat/components/chartElement/__stories__/chart.mdx

```diff
@@ -46,13 +38,10 @@ import '@carbon-labs/ai-chat-chart/es/index.js';
 ## Implementation
 
 ### &lt;clabs-chat-chart&gt; attributes and properties
-
-<ArgTypes of="clabs-chat-chart" />
+<Controls story="Playground" />
```

Comment:
Should we stick to `<ArgTypes of="clabs-chat-chart" />` here? With that, it displays all the attributes associated with chart component

Reply:
actually nvm, I see both `ArgTypes` and `Control` do similar things

Reply:
I did think showing the attributes was weird but didn't see a difference when changing, I'll try again and push